config/config.go

```diff
@@ -152,16 +213,36 @@ func (r *Regexpwrapper) UnmarshalText(text []byte) error {
 	return err
 }
 
-// CheckNames return list of plugin.checks._name_
+// CheckNames returns a list of name of the check plugins
 func (conf *Config) CheckNames() []string {
 	checks := []string{}
-	for name := range conf.Plugin["checks"] {
+	for name := range conf.CheckPlugins {
 		checks = append(checks, name)
 	}
 	return checks
 }
 
-// LoadConfig XXX
+// CustomIdentifiers returns a list of customIdentifiers.
```

Comment:
This comment, or method name should be more verbose that this method aggregates information inside `config.MetricPlugins`. We cannot read it from current method name`CustomIdentifiers`.